### PR TITLE
[xbmc][settings][GL][DX]: Remove test patterns

### PR DESCRIPTION
**Pull request by VelocityRa**
## Description

Removes the **Test Patterns** available in **System Settings->Display->Calibration** in the "Expert" category.

## Motivation and Context

It's unused by most people and unmaintained (only DX works as intended).
Discussed in https://github.com/xbmc/xbmc/issues/14723#issuecomment-433436188

## How Has This Been Tested?

Option in Settings is not there anymore.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/guilib/WindowIDs.dox [hunk 1]
```diff
@@ -14,7 +14,6 @@ This page shows the window names, the window definition, the window ID and the s
 | FileManager             | WINDOW_FILES                         | 10003     | FileManager.xml                     |
 | Settings                | WINDOW_SETTINGS_MENU                 | 10004     | Settings.xml                        |
 | SystemInfo              | WINDOW_SYSTEM_INFORMATION            | 10007     | SettingsSystemInfo.xml              |
-| TestPattern             | WINDOW_TEST_PATTERN                  | 10008     | none                                |
```
Comment (MartijnKaijser) [inline xbmc/guilib/WindowIDs.dox]: We should probably be starting to add this to the changelog
See https://github.com/xbmc/xbmc/pull/14542 as example

Comment (VelocityRa) [reply xbmc/guilib/WindowIDs.dox]: That PR is not merged yet though, there's no changelog atm.

**Review by afedchin — COMMENTED** [review]
I haven't opinion about needing of the feature but the changes looks good

**Review by lrusak — APPROVED** [review]
Changes look fine to me!

**Comment by MartijnKaijser** [comment]
Any objections on doing this for v18 cause it's a non functional part we remove?

**Comment by afedchin** [comment]
I have no objections

**Pull request merged**